repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publication.ftl

```diff
@@ -138,6 +138,38 @@
                 </p>
             </#if>
 
+            <#if publication.bodySections?has_content>
```

Comment:
This section is quite elaborate and it feels that it would be easier to read better if it could be converted into a few macros.

How about:
* one macro to render 'image' section, embedding the `<#if section.sectionType == 'image` check
* one macro to render 'text' section, embedding the `<#if section.sectionType == 'text` check
* top level one to be called from the template, to call all the other macros, one after another. Since individual, specialised macros do the `sectionType` checks themselves, the top-level one wouldn't even have to implement any, just call all the macros, blindly.